forum/projects/views.py

```diff
@@ -1,3 +1,80 @@
+from asgiref.sync import async_to_sync
+from channels.layers import get_channel_layer
+from django.shortcuts import get_list_or_404
 from django.shortcuts import render
 
-# Create your views here.
+from rest_framework import viewsets, permissions
+
+from rest_framework_simplejwt.authentication import JWTAuthentication
+
+from .models import Project, Investor, ProjectSubscription
+from .serializers import ProjectSerializer, HistoricalProjectSerializer
+from .permissions import UpdateOwnProject
+
+from startups.models import Startup
+from forum.utils import get_changed_fields, build_email_message
+from forum.tasks import send_email_task
+
+
+class ProjectViewSet(viewsets.ModelViewSet):
+    """
+    A viewset for viewing and editing Project instances.
+    """
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+    permissions_classes = (permissions.IsAuthenticated, UpdateOwnProject)
```

Comment:
It's not obligatory to declare `authentication_classes = (JWTAuthentication,)` because we have defined that in forum/settings.py.